packages/api-kit/tests/e2e/getSafesByOwner.test.ts

```diff
@@ -14,14 +15,14 @@ describe('getSafesByOwner', () => {
   })
 
   it('should fail if owner address is empty', async () => {
-    const ownerAddress = ''
+    const ownerAddress = '' as Address
     await chai
       .expect(safeApiKit.getSafesByOwner(ownerAddress))
       .to.be.rejectedWith('Invalid owner address')
   })
 
   it('should fail if owner address is not checksummed', async () => {
-    const ownerAddress = '0x9cCBDE03eDd71074ea9c49e413FA9CDfF16D263B'.toLowerCase()
+    const ownerAddress = '0x9cCBDE03eDd71074ea9c49e413FA9CDfF16D263B'.toLowerCase() as Address
```

Comment:
These toLowerCase can be avoided by using the getAddress from viem that checksum the addresses and return an `0xstring`